katharsis-jpa/src/main/java/io/katharsis/jpa/JpaModule.java

```diff
@@ -404,25 +406,40 @@ private void setupServerRepositories() {
 	private void setupMappedRepository(MappedRegistration<?, ?> mapping) {
 		MetaEntity metaEntity = metaLookup.getMeta(mapping.getEntityClass()).asEntity();
 		if (isValidEntity(metaEntity)) {
-			JpaEntityRepository<?, ?> repository = repositoryFactory.createEntityRepository(this, mapping.getDtoClass());
+			QuerySpecResourceRepository<?, ?> repository = filterCreation(
+					repositoryFactory.createEntityRepository(this, mapping.getDtoClass()));
 			context.addRepository(mapping.getDtoClass(), repository);
-
 			setupRelationshipRepositories(mapping.getDtoClass());
 		}
 	}
 
+	private QuerySpecResourceRepository<?, ?> filterCreation(JpaEntityRepository<?, ?> repository) {
+		QuerySpecResourceRepository<?, ?> filteredRepository = repository;
+		for (JpaRepositoryFilter filter : filters) {
+			filteredRepository = filter.filterCreation(repository);
+		}
+		return filteredRepository;
+	}
+
+	private QuerySpecRelationshipRepository<?, ?, ?, ?> filterCreation(JpaRelationshipRepository<?, ?, ?, ?> repository) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this unused private "filterCreation" method. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AUnusedPrivateMethod)